index.js

```diff
@@ -1,44 +1,20 @@
 const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout); function ask(questionText) {
-    return new Promise((resolve, reject) => {
-        rl.question(questionText, resolve);
-    });
-}
-function randomInteger(min, max) {
-    let range = max - min + 1;
-    return min + Math.floor(Math.random() * range);
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
 }
+
 start();
-function getRandomInt(max) {
-    return Math.floor(Math.random() * Math.floor(max));
-}
+
 async function start() {
-    console.log("Let's play a game where you (human) make up a number between 1 and 100 and I (computer) try to guess it.")
-    let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-    console.log('You entered: ' + secretNumber);
-    userResponse = secretNumber
-    let guess = getRandomInt(100)
-    userResponse = await ask("Is " + guess + " your number? ")
-    let maxNumber = 100
-    let minNumber = 1
-    while (guess !== secretNumber) {
-        if (guess > secretNumber) {
```

Comment:
Instead of using the secret number to determine if the guess is too high or too low, you should have the player tell the computer if it's higher or lower.  Pretend you don't have access to `secretNumber`